### ISSUE TITLE
gluster blockd status should be checked in liveness/readiness probe

### ISSUE THREAD

**Issue by raghavendra-talur** [gluster/gluster-kubernetes#394]
Gluster container also has gluster-blockd service running. In some cases, user might want pod to fail readiness probe if gluster blockd is not running. This can be achieved by making a script as readiness checker which has checks in the following way
```
ret = 0
if glusterd not running
   ret = -1
if gluster-blockd not running AND ENV[blockd is must]==yes
   ret = -1
return ret
```

**Comment by jarrpa** [comment]
@raghavendra-talur Do you think we should do that in the pod spec or implement it as a script in the container?

**Comment by raghavendra-talur** [comment]
@jarrpa script in the container. pod spec should only refer to the script.

**Comment by nixpanic** [comment]
This has been addressed through #523.
